tests/tests.cc

```
@@ -780,6 +781,464 @@ class Tests {
+                                     std::to_string(stat_resp.size));
+          }
+        }
+
+        // async GetObject
+        {
+          std::string content;
+          minio::s3::GetObjectArgs gargs;
+          gargs.bucket = bucket_name_;
+          gargs.object = object_name;
+          gargs.datafunc =
+              [&content](minio::http::DataFunctionArgs args) -> bool {
```

Cppcheck (2.21.0)

[performance] Function parameter 'args' should be passed by const reference.

(passedByValue)